Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/JoiSchemaEnhancer.ts

```diff
@@ -29,6 +31,18 @@ const descriptorSchema = Joi.object({
   description: Joi.string().optional(),
 });
 
+/**
+ * Returns a new PropertyModifier that is the concatenation of two. Used for Commons and sub-Commons,
+ * where there is a chain of parent modifiers that cannot be completely pre-computed
+ * (without a different design, like pre-computing all possible paths).
+ */
+function propertyModifierConcat(p1: PropertyModifier, p2: PropertyModifier): PropertyModifier {
+  return {
```

Comment:
PropertyModifier tracks property naming modifications that occur as you go from parent entity -> property. This is to bridge the modifiers when there are multiple entities involved e.g. entity -> common property -> common entity -> other property